ingest/profiles/default/config.yaml

```diff
@@ -0,0 +1,3 @@
+cores: all
+printshellcmds: true
+ri: true
```

Comment:
Do you know you can make these your global default by putting the profile somewhere shared and setting `SNAKEMAKE_PROFILE`, e.g.

```
export SNAKEMAKE_PROFILE=~/.config/snakemake/default
```